keel-retrofit/src/main/kotlin/com/netflix/spinnaker/keel/retrofit/SpinnakerCall.kt

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.keel.retrofit
+
+import java.lang.annotation.Documented
+import java.lang.annotation.Retention
+import java.lang.annotation.RetentionPolicy.RUNTIME
+import kotlin.annotation.AnnotationTarget.FUNCTION
+import kotlin.annotation.AnnotationTarget.PROPERTY_GETTER
+import kotlin.annotation.AnnotationTarget.PROPERTY_SETTER
+import retrofit2.http.Url
+
+/** Marks a retrofit service request function as a Spinnaker call. */
+@Documented
+@Target(FUNCTION, PROPERTY_GETTER, PROPERTY_SETTER)
+@Retention(RUNTIME)
+annotation class SpinnakerCall(
```

Comment:
This is just a marking interface so we can tag which requests we want to wrap with the call adapter.